backend/src/graphql/resolver/ContributionMessageResolver.ts

```diff
@@ -158,16 +158,18 @@ export class ContributionMessageResolver {
       contributionMessage.isModerator = true
       await queryRunner.manager.insert(DbContributionMessage, contributionMessage)
 
-      if (
-        contribution.contributionStatus === ContributionStatus.DELETED ||
-        contribution.contributionStatus === ContributionStatus.DENIED ||
-        contribution.contributionStatus === ContributionStatus.PENDING
-      ) {
-        contribution.contributionStatus = ContributionStatus.IN_PROGRESS
-        await queryRunner.manager.update(DbContribution, { id: contributionId }, contribution)
-      }
-
       if (messageType !== ContributionMessageType.MODERATOR) {
+        // change status (does not apply to moderator messages)
```

Comment:
Good catch. Please add a unit test for this, e.g. test that a moderator message does not change the status